Use OpenEx when available with visOpenCopy and visOpenRO

agaer opened the pull request:
Using OpenEx with visOpenCopy and visOpenRO allows visio2img to open a vsd
even if another visio instance has the file already opened.

tk0miya commented on visio2img/visio2img.py:
```diff
@@ -76,7 +76,16 @@ def open(self, filename):
+                # if they're open in another visio instance...
+                visOpenCopy = 0x1
+                visOpenRO = 0x2
+                open_flags = visOpenCopy + visOpenRO
```
It would be better to use logical OR operator.

tk0miya commented:
Sorry for confusing you. I fixed test script just now, and then this PR passes all tests.

I cannot check the behavior because I lost virtual machines for testing this.
But this PR looks good. so I will merge this after fixing nits :-)

agaer committed:
Use OpenEx when available with visOpenCopy and visOpenRO flags

Using OpenEx with visOpenCopy and visOpenRO flags allows visio2img to open
a vsd even if another visio instance has the file already opened.

agaer commented:
Ok, will change...
